src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,17 +30,26 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'position_after_functions_and_oop_constructs' => self::NEXT_LINE,
```

Comment:
Yes, but for me boolean implies on – off behaviour. This is more of a A – B situation.